src/status_line.ts

```diff
@@ -1,31 +1,28 @@
-import { Disposable, StatusBarAlignment, StatusBarItem, window, workspace } from "vscode";
+import { Disposable, StatusBarAlignment, StatusBarItem, window } from "vscode";
+
+import { config } from "./config";
 
 // !Maybe we can support &statusline
 
 export class StatusLineController implements Disposable {
     private _modeText = "";
     private _statusText = "";
     private _msgText = "";
-    private _seperator = " - ";
 
     private statusBar: StatusBarItem;
 
     public constructor() {
         this.statusBar = window.createStatusBarItem(StatusBarAlignment.Left, 10);
-        this._seperator = workspace.getConfiguration("window").get("titleSeparator", this._seperator);
+        this.statusBar.show();
     }
 
     private refreshStatusBar() {
         const items = [];
         this._modeText.length && items.push(this._modeText);
         this._statusText.length && items.push(this._statusText);
         this._msgText.length && items.push(this._msgText);
-        if (items.length) {
-            this.statusBar.text = items.join(this._seperator);
-            this.statusBar.show();
-        } else {
-            this.statusBar.hide();
-        }
+        // Always show the status line
```

Comment:
+1 for this also. It was weird that the statusbar disappears.

Reply:
I think we need to adjust the priority. It looks strange right now. It should be displayed in the middle so that it doesn't cause other items to move around.

![image](https://github.com/vscode-neovim/vscode-neovim/assets/47070852/f4133bc7-087e-4db1-8d77-d5116b7da23a)

Reply:
Should use -1